code/modules/clothing/glasses/glasses.dm

```diff
@@ -8,6 +8,7 @@
 	/// The amount of nightvision these glasses have. This should be a number between 0 and 1.
 	var/lighting_alpha = LIGHTING_PLANE_ALPHA_VISIBLE
 	var/invisa_view = FALSE
+	var/eye_tint = FALSE //Used for photosensitivity.
```

Comment:
variable name does not accurately represent what it's used in: eye_tint makes me think it tints the eyes like thermals.

```suggestion
	var/sensitivity_protection = FALSE //Used for photosensitivity.
```

If anything it feels like this variable is completely unnecessary and you should just give EYE_PROTECTION_FLAVOR to the glasses affected. If you don't want to do that, make a new tier in-between flavor called EYE_PROTECTION_WEAK_FLAVOR with a value of 0.5 or something.